dm/worker/config.go

```diff
@@ -246,25 +235,34 @@ func (c *Config) UpdateConfigFile(content string) error {
 
 // Reload reload configure from ConfigFile
 func (c *Config) Reload() error {
-	var pswd string
-	var err error
-
 	if c.ConfigFile == "" {
 		c.ConfigFile = "dm-worker-config.bak"
 	}
 
-	err = c.configFromFile(c.ConfigFile)
+	err := c.configFromFile(c.ConfigFile)
 	if err != nil {
 		return errors.Trace(err)
 	}
 
-	if len(c.From.Password) > 0 {
-		pswd, err = utils.Decrypt(c.From.Password)
+	return nil
+}
+
+// DecryptPassword return a decrypted config replica in config
+func (c *Config) DecryptPassword() (*Config, error) {
+	// try decrypt password for To DB
```

Comment:
why add this comment? in fact, it's **From** DB?

Reply:
remove it